### ISSUE TITLE
Input generator on 4K screens

### ISSUE THREAD

**Issue by cmbant** [CobayaSampler/cobaya#41]
I have the same issue in GetDistGUI: on 4K Windows screen with PySide2 things are too small

![cobaya_4K](https://user-images.githubusercontent.com/3958819/65760920-ca883e00-e115-11e9-8721-a67869df1247.png)

I don't know what the solution is, no permutation of the following seems to give consistent results for GetDistGUI
```
 QApplication.setAttribute(Qt.AA_EnableHighDpiScaling) # DPI support
 QCoreApplication.setAttribute(Qt.AA_UseHighDpiPixmaps) # HiDPI pixmaps
 os.environ["QT_AUTO_SCREEN_SCALE_FACTOR"] = "1"
 os.environ["QT_SCALE_FACTOR"] = "2" 
```

**Comment by cmbant** [comment]
Explicitly setting the window font using setStyleSheet("* {font-size:8pt;}") may fix this.

**Comment by cmbant** [comment]
Should be fixed in https://github.com/CobayaSampler/cobaya/commit/667cb0de9037f25f2c1c7c3c1cdadc9e029e5841 at least for Windows. Worth checking on other systems.

**Comment by JesusTorrado** [comment]
Looks tiny in my system (Kubuntu 18.04 with Plasma 5.12). Maybe Windows and Linux scale fonts and window elements in two different ways? Investigating...

**Comment by cmbant** [comment]
Interesting, looks OK on the Ubuntu cosmobox VM

**Comment by cmbant** [comment]
Is yours high DPI? Does 10pt or 12pt looks OK? If so might as well just use that.

**Comment by JesusTorrado** [comment]
Somewhat high DPI. Qt most definitely handles fractional scale factors (my desktop interface is based on Qt and does. In any case, the correct strategy, that should work if there are no bugs, would be to let the app get from the OS both the font size for the different widgets (in your case, in the first pic, it apparently worked for the menu but not for the drop-down selectors) and also the scaling factor.

I am going to branch out, fix some hard-coded numbers I put there in the past (fixed pixel sizes), remove any mention to a font size, and let's try to go on from there, if you don't mind.

**Comment by JesusTorrado** [comment]
Fixed by #45
